cmd/containerd-shim-runhcs-v1/pod.go

```diff
@@ -84,6 +85,16 @@ func createPod(ctx context.Context, events publisher, req *task.CreateTaskReques
 	owner := filepath.Base(os.Args[0])
 	isWCOW := oci.IsWCOW(s)
 
+	templateID := oci.ParseAnnotationsTemplateID(ctx, s)
+	isTemplateCreation := oci.ParseAnnotationsSaveAsTemplate(ctx, s)
+	var utc *uvm.UVMTemplateConfig
```

Comment:
nit: I feel like if a variable is not being used in close proximity then it should have a more descriptive name.